packages/core/src/components/list-slider/list-slider.tsx

```diff
@@ -213,13 +225,19 @@ export class AtomListSlider {
     this.handleSwipe()
   }
 
+  handleTouchMove = () => {
+    this.isTouchMoved = true
+  }
+
   handleSwipe = () => {
-    const lastIndex = this.sliderItems.length - 1
-    const isSwipeLeft = this.touchEndX < this.touchStartX
+    if (this.isTouchMoved) {
```

Comment:
Suggestion: 

`if(!this.isTouchMoved) return`

Reply:
1667a9491b45969c94741db11592dcd49084c3c7